### ISSUE TITLE
clear screen and /or scrollbar not working 

### ISSUE THREAD

**Issue by zangetsudo** [kingToolbox/WindTerm#447]
hi, there.

we have tried with ctrl+l, ctrl+shift+l, nothing clear, but one/or two new line is added to the last screen.
how can we clean the whole screen, we checked with menu, key boards shutcut , nothings.

regards, zan

**Comment by dmberrycom** [comment]
Same issue here.

Clear Screen via menu or shortcut CTRL+L does not function, tested on shell and serial sessions, and both exhibit the same behaviour.

**Comment by zangetsudo** [comment]
"clear scrollbar" works now after we update to WindTerm 2.3.0 Prerelease.
but the current screen cannot be clear anyway.

@dmberrycom

**Comment by kingToolbox** [comment]
Sorry for the late replay.

Can you tell me which `System` of the session you selected is? Only `linux`, `cmd`, `powershell` and `routeros` support `clear screen`. In addition, can `Ctrl+L` clear the screen normally in other terminals?

**Comment by dmberrycom** [comment]
Yes, these are serial ports. I've tried changing the system type, but this makes no difference.

Ctrl+L does not clear in this use.

![image](https://user-images.githubusercontent.com/6107912/150199230-d16d0dd1-228d-43d3-a1d1-7e6d876743ec.png)

**Comment by kingToolbox** [comment]
`Ctrl+L` needs the support of the remote system. After pressing `Ctrl+L`, `WindTerm` will send the text of `clearScreen` configured in the `.system` file.

Taking  the `linux` system as an example, the `clearScreen` is defined as `\u000c` in the `WindTerm_install_path/terminal/schemes/linux.system`. So when you press `Ctrl+L`, `WindTerm` will send `\u000c` to the remote system. If the remote system understands this command, it will send the control sequence `ED` to `WindTerm` to clear the screen, and then `WindTerm` will clear the screen after receiving `ED`. In this way, the remote and local can be guaranteed to be exactly the same.

So, does your serial device support clear screen? You can try other terminals, if other terminals can clear the screen, please let me know and I will tell you how to solve it, thank you.

**Comment by dmberrycom** [comment]
No, I've been using windterm to connect to very dumb devices that just output diagnostic data via a serial port. The devices are not actually terminals just basic USART devices connected over USART to serial adapters. I was looking for the ability just to clear the output on the screen, for example when rebooting a device to clearly view the new output.

Hope this makes sense.

**Comment by kingToolbox** [comment]
Hm, dumb devices usually cannot support clearing the screen. However, if you have a way to send `\033[2J` to `WindTerm` from the serial port, you can also clear the screen, just like executing `echo -e "\033[2j"` in the `Linux` system.

**Comment by kingToolbox** [comment]
SORRY! `WindTerm_2.3.0_Prerelease_2` and `WindTerm_2.3.0_Prerelease_3` has a **serious SFTP bug** #487 and may cause file corruption after **uploaded**.

Please use the old versions or update to the [WindTerm_2.3.0_Prerelease_4](https://github.com/kingToolbox/WindTerm/releases/tag/2.3-prerelease) version. Thank you.

**Comment by kingToolbox** [comment]
Duplicate of #335 

In the new [WindTerm_2.4.0_Prerelease_1](https://github.com/kingToolbox/WindTerm/releases/tag/2.4-prerelease), a new `dumb` system has been added. In addition to features such as highlighting and folding, the `dumb` system supports clearing the screen locally. So for some `dumb` devices, especially `serial` devices, if you need to clear the screen locally, please use `dumb` system instead of `none` system.

The `dumb` system also supports `->` command prompt for the `VxWorks` system. Please download and use it, if you have any questions, please let me know, thank you.

BTW:
**Please use `dumb` system with caution and only for the `dumb` devices.** If the remote supports the `clear screen command`, using `dumb` system to clear the screen locally will cause the local display to be out of sync with the remote, which may cause various display errors.
